src/objects.rs

```diff
@@ -330,7 +330,29 @@ pub struct WebhookInfo {
 
     #[serde(skip_serializing_if = "Option::is_none")]
     #[builder(setter(into, strip_option), default)]
-    pub allowed_updates: Option<Vec<String>>,
+    pub allowed_updates: Option<Vec<AllowedUpdate>>,
+}
+
+/// Control which updates to receive.
+/// Specify an empty list to receive all update types except `ChatMember`.
+/// [Official documentation](https://core.telegram.org/bots/api#getupdates).
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
```

Comment:
No need for references with `Copy`
```suggestion
#[derive(Debug, Clone, Copy, Serialize, Deserialize, PartialEq)]
```